public/static/js/events.js

```diff
@@ -0,0 +1,17 @@
+var addSelectorsAndOwnerType = function(subscription, type, id) {
```

Comment:
1. Is there any reason you used this syntax vs `function addSelectorsAndOwnerType(...)`? I'd prefer an accidental assignment to not be able to change behaviour everywhere.

2. Does it make more sense to put this in the subscriptions.js file?

Reply:
Bad habit. Moved it into subscriptions.js